lib/imgix/path.rb

```diff
@@ -132,16 +132,19 @@ def method_missing(method, *args, &block)
     # This includes " +?:#" characters. If a path is being used as a proxy, utf8
     # encode everything. If it is not being used as proxy, leave certain chars, like
     # "/", alone. Method assumes path is not already encoded.
-    def sanitize_path(path)
+    def sanitize_path(path, options = {})
       # remove the leading "/", we'll add it back after encoding
       path = path.slice(1, path.length) if Regexp.new('^/') =~ path
-      # if path is being used as a proxy, encode the entire thing
-      if /^https?/ =~ path
-        return encode_URI_Component(path)
-      else
-        # otherwise, encode only specific characters
-        return encode_URI(path)
+      if !options[:disable_path_encoding]
+        # if path is being used as a proxy, encode the entire thing
+        if /^https?/ =~ path
+          return encode_URI_Component(path)
+        else
+          # otherwise, encode only specific characters
+          return encode_URI(path)
+        end
       end
+      return "/" + path
```

Comment:
Nit, but I think it's a little easier to read if we remove the nested conditionals here and use `elsif` instead.

```suggestion
      if options[:disable_path_encoding]
        return "/" + path
      # if path is being used as a proxy, encode the entire thing
      elsif /^https?/ =~ path
        return encode_URI_Component(path)
      else
        # otherwise, encode only specific characters
        return encode_URI(path)
      end
```